Headers/CoreGraphics/CGContext.h

```diff
@@ -562,6 +578,19 @@ CGRect CGContextConvertRectToDeviceSpace(CGContextRef ctx, CGRect rect);
 
 CGRect CGContextConvertRectToUserSpace(CGContextRef ctx, CGRect rect);
 
+bool CGContextPathContainsPoint(CGContextRef c,
+    CGPoint point, CGPathDrawingMode mode);
+
+/* CoreGraphics Private APIs */
```

Comment:
Well, I selected line 584 when I was creating the commit, not my fault that GitHub has... flaws. :smile:

I was referring to all the APIs below that were marked as private. I did not check any to be private except `CGContextSetCTM()`